react/src/hooks/unlinked/useUnlinkedFilesQuery.tsx

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from "react-query";
+import { basicFetch } from "../utils";
+
+async function fetchFiles() {
+    return await basicFetch("/api/unlinked");
+}
+
+/**
+ * Return result of GET /api/unlinked.
+ *
+ * That is, return a sorted list of filenames
+ * for all unlinked files in MinIO.
+ */
+export function useUnlinkedFilesQuery() {
+    const result = useQuery<string[], Response>("unlinked", fetchFiles);
+    if (result.isSuccess)
+        result.data = result.data.sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()));
```

Comment:
Should we sort on the backend instead and the frontend is only responsible for keeping its own representation sorted?

Reply:
Yeah that might be a good idea. I don't think there's any frontend actions that add to unlinked files, only remove, so no extra care needs to be taken to keep the list sorted.

Reply:
Some frontend actions do add to the list but care was already taken then to sort the list before the state is updated. You can just call `.sort()` or `sorted` on the list before it's passed to `jsonify` in the endpoint.